apps/web/src/constants/routes.ts

```diff
@@ -49,15 +49,20 @@ export enum ROUTES {
   ORGANIZATION = '/settings/organization',
   SECURITY = '/settings/security',
   BILLING = '/settings/billing',
-  /** Novu V2 routes */
+
+  /** Novu V2 routes, defined as nested routes */
   STUDIO = '/studio',
-  STUDIO_FLOWS = '/studio/flows',
-  STUDIO_FLOWS_VIEW = '/studio/flows/:templateId',
-  STUDIO_ONBOARDING = '/studio/onboarding',
-  STUDIO_ONBOARDING_PREVIEW = '/studio/onboarding/preview',
-  STUDIO_ONBOARDING_SUCCESS = '/studio/onboarding/success',
-  STUDIO_FLOWS_STEP_EDITOR = '/studio/flows/:templateId/step/:stepId',
-  STUDIO_FLOWS_TEST = '/studio/flows/:templateId/test',
+  STUDIO_FLOWS = 'flows',
+  STUDIO_FLOWS_VIEW = 'flows/:templateId',
+  STUDIO_FLOWS_STEP_EDITOR = 'flows/:templateId/step/:stepId',
+  STUDIO_FLOWS_TEST = 'flows/:templateId/test',
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** I kept `templateId` for simplicity when I was first scaffolding, but we should probably change this to something more "modern" within our terminology. `workflowId`?‏